include/arch/arm/cortex_m/mpu/arm_core_mpu.h

```diff
@@ -20,7 +20,7 @@ extern "C" {
  * MPU driver. The function is meant to be invoked once,
  * during system initialization.
  */
-void z_arch_configure_static_mpu_regions(void);
+void arch_configure_static_mpu_regions(void);
```

Comment:
Could you see if these can be cleaned up? At some point I might have thought that this api would be cross-ARCH but it does not seem to be the case...

Reply:
I think the z_arm_ already exists, and is under arch/arm/core/cortex-m

Reply:
I think the z_arm_ already exists, and is under arch/arm/core/cortex-m

Reply:
I think the z_arm_ already exists, and is under arch/arm/core/cortex-m

Reply:
broken record?

Reply:
> broken record?

Something with my browser/GH account I guess. Sorry for the noise